### PR TITLE
APPS-1126 locked reorg support

### PR DESCRIPTION
**Pull request by mhrvol**
Removed reorg/locked limitation. No issues were found. I did not test custom applet. Test cases that passed are in the PR

### CODE REVIEW

**Comment by commandlinegirl** [comment]
@mhrvol - @YuxinShi0423  has changed the name of the workflow `trains_station.wdl` in her recent merge, so it may have affected the test. I merged `develop` into your branch but it still fails. Perhaps something needs to be updated in the Unit Test assertion?

**Comment by YuxinShi0423** [comment]
Hi @mhrvol @commandlinegirl I don't think I have changed the name of the test... I just used another wdl file in another unit test.

File: doc/CustomReorgAppletExample.md [hunk 1]
```diff
@@ -32,8 +34,8 @@ simple_task {
 The workflow will generate an array of files - `out.vcf` and `out.bam`.
 Suppose we want to reorganize the results into to 2 different folders on the project.
 
-`out.vcf` will go to /results/out/vcf
-`out.bam` will go to /bam/
+`out.vcf` will go to /folder_for_bam
+`out.bam` will go to /folder_for_vcf
```
Comment (Gvaihir) [inline doc/CustomReorgAppletExample.md]: Vice versa?

Comment (mhrvol) [reply doc/CustomReorgAppletExample.md]: Changed. thanks.

**Comment by Gvaihir** [comment]
@mhrvol - can you decide if this has to be merged or closed?

**Pull request closed without merging**